sdk/python/lib/pulumi/automation/_workspace.py

```diff
@@ -30,7 +30,7 @@ class StackSummary:
 
     name: str
     current: bool
-    update_in_progress: bool
+    update_in_progress: Optional[bool]
```

Comment:
This is a bit awkward, since it's a backwards incompatible change, but I'm not sure how to get around that.  I guess we could still make it `false` instead of `None` if it's not in the json, and then change it in the 4.0 major release?  I'd be curious to hear if anyone has thoughts on this.

And similar story with the nodejs SDK.

Reply:
I think it's probably fine, it's an easy break to fix by just doing a coalesce with false.